tests/modelbench_tests/test_run_journal.py

```diff
@@ -226,15 +228,16 @@ def test_item_exception_entry(self, journal, sut):
         assert e["message"] == "fail"
         assert e["test"] == "a_test"
         assert e["prompt_id"] == "id1"
-        assert e["sut"] == tri.sut.uid
         assert e["exception"]["class"] == "ValueError"
+        assert "sut" not in e  # we no longer output the sut at the item level
 
     def make_test_run_item(self, source_id, test_id, text):
-        from modelbench.benchmark_runner import TestRunItem, ModelgaugeTestWrapper
-        from modelbench_tests.test_benchmark_runner import AFakeTest
+        from modelbench.benchmark_runner import ModelgaugeTestWrapper, TestRunItem
```

Comment:
Could you please set your autoformatter to not change lines you didn't edit and/or manually reformat?